README.md

```diff
@@ -239,20 +239,40 @@ $ semver 1.2.4-beta.0 -i prerelease
 
 The method `.inc` takes an optional parameter 'identifierBase' string
 that will let you let your prerelease number as zero-based or one-based.
+If you provide false zero will be omitted.
 If you do not specify this parameter, it will default to zero-based.
 
 ```javascript
 semver.inc('1.2.3', 'prerelease', 'beta', '1')
 // '1.2.4-beta.1'
 ```
 
+```javascript
+semver.inc('1.2.3', 'prerelease', 'beta', false)
+// '1.2.4-beta'
+```
+
 command-line example:
 
 ```bash
 $ semver 1.2.3 -i prerelease --preid beta -n 1
 1.2.4-beta.1
 ```
 
+```bash
+$ semver 1.2.3 -i prerelease --preid beta -n false
+1.2.4-beta
+```
+
+
+if another version is created with same identifier then the prerelease
+number is used based on Prerelease Identifier Base
```

Comment:
This is confusing, what does it mean?

Reply:
I think perhaps for now the code example tells a better story than any comment can.

Reply:
I accidentally left outdated paragraph from initial commit, is this better